files/build_templates/sonic_debian_extension.j2

```diff
@@ -235,7 +235,7 @@ sudo bash -c "echo '{ \"DEVICE_METADATA\": { \"localhost\": { \"default_bgp_stat
 sudo cp $IMAGE_CONFIGS/snmp/snmp.yml $FILESYSTEM_ROOT/etc/sonic/
 
 # Copy ASN configuration files
-sudo cp $IMAGE_CONFIGS/asn/deployment_id_asn_map.yml $FILESYSTEM_ROOT/etc/sonic/
```

Comment:
Hi @pavel-shirshov 
I think this change breaks bgp_speaker TCs.
```
TASK [test : fetch bgp speaker asn number from DUT] ****************************
Monday 11 November 2019  12:07:39 +0000 (0:00:00.110)       0:00:12.550 ******* 
[0;31mfatal: [cab18-2-dut]: FAILED! => {"changed": true, "cmd": "sonic-cfggen -m -d -y /etc/sonic/deployment_id_asn_map.yml -v \"deployment_id_asn_map[DEVICE_METADATA['localhost']['deployment_id']]\"", "delta": "0:00:00.166444", "end": "2019-11-11 12:07:39.972509", "failed": true, "rc": 1, "start": "2019-11-11 12:07:39.806065", "stderr": "Traceback (most recent call last):\n  File \"/usr/local/bin/sonic-cfggen\", line 318, in <module>\n    main()\n  File \"/usr/local/bin/sonic-cfggen\", line 250, in main\n    with open(yaml_file, 'r') as stream:\nIOError: [Errno 2] No such file or directory: '/etc/sonic/deployment_id_asn_map.yml'", "stdout": "", "stdout_lines": [], "warnings": []}[0m
```
Could you please check?

Reply:
Hi @akokhan 
Thank you for your report. Can you please check the fix? https://github.com/Azure/sonic-mgmt/pull/1204 ?